lib/pltr/v2/actions/ui.js

```diff
@@ -189,3 +191,11 @@ export function selectCharacterAttributeBookTab(bookId) {
 export function selectCharacter(id) {
   return { type: SELECT_CHARACTER, id }
 }
+
+export function setCardDialogOpen(cardId, beatId, lineId) {
+  return { type: SET_CARD_DIALOG_OPEN, cardId, lineId, beatId }
```

Comment:
This is where you *might* use `redux-thunk` to check whether the card, beat and line actually exist before dispatching the action.